iosHyperskillApp/iosHyperskillApp/Sources/Modules/Streak/Views/StreakFreezeModalViewController.swift

```diff
@@ -0,0 +1,260 @@
+        let image = UIImage(named: streakFreezeState.image)?.withRenderingMode(.alwaysOriginal)
+        let imageView = UIImageView(image: image)
+
+        contentStackView.addArrangedSubview(imageView)
+
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.snp.makeConstraints { make in
+            make.width.equalTo(view).multipliedBy(appearance.imageViewSizeRatio.width)
+            make.height.equalTo(view).multipliedBy(appearance.imageViewSizeRatio.height)
+        }
+    }
+
+    private func setupTitleView() {
+        let label = UILabel()
+        label.text = streakFreezeState.title
+        label.font = .preferredFont(forTextStyle: .largeTitle, compatibleWith: .init(legibilityWeight: .bold))
+        label.textColor = .primaryText
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = 0
+
+        contentStackView.addArrangedSubview(label)
+    }
+
+    private func setupStatusStackView() {
+        let containerStackView = UIStackView()
+        containerStackView.axis = .horizontal
+        containerStackView.distribution = .equalSpacing
+
+        contentStackView.addArrangedSubview(containerStackView)
+        containerStackView.translatesAutoresizingMaskIntoConstraints = false
+        containerStackView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+        }
+
+        containerStackView.addArrangedSubview(makeStatusBadgeView())
+
+        let statusTextLabel = UILabel()
+        statusTextLabel.font = .preferredFont(forTextStyle: .subheadline)
+        statusTextLabel.textColor = .primaryText
+        statusTextLabel.text = streakFreezeState.statusText
+        containerStackView.addArrangedSubview(statusTextLabel)
+    }
+
+    private func makeStatusBadgeView() -> UIStackView {
+        let containerStackView = UIStackView()
+        containerStackView.axis = .horizontal
+        containerStackView.spacing = appearance.statusBadgeSpacing
+
+        let image = UIImage(
+            named: streakFreezeState.statusIcon
+        )?.withRenderingMode(.alwaysOriginal)
+        let imageView = UIImageView(image: image)
+
+        containerStackView.addArrangedSubview(imageView)
+
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.snp.makeConstraints { make in
+            make.height.equalTo(appearance.statusBadgeImageWidthHeight)
+            make.width.equalTo(appearance.statusBadgeImageWidthHeight)
+        }
+
+        let label = UILabel()
+        label.text = String(streakFreezeState.statusCount)
+        label.font = .preferredFont(forTextStyle: .title3)
+        label.textColor = .primaryText
+        containerStackView.addArrangedSubview(label)
+
+        return containerStackView
+    }
+
+    private func setupTextView() {
+        let label = UILabel()
+        label.text = Strings.Streak.FreezeModal.text
+        label.font = .preferredFont(forTextStyle: .headline)
```

Comment:
```suggestion
        label.font = .preferredFont(forTextStyle: .body)
```

`headline` имеет только слово `automatically`, можно забить на это